app/assets/javascripts/discourse/app/routes/wizard.js

```diff
@@ -1,31 +1,31 @@
 import Route from "@ember/routing/route";
-import { findWizard } from "wizard/models/wizard";
+import Wizard from "discourse/static/wizard/models/wizard";
 import DisableSidebar from "discourse/mixins/disable-sidebar";
 
-export default Route.extend(DisableSidebar, {
+export default class WizardRoute extends Route.extend(DisableSidebar) {
```

Comment:
I kept the mixin usage here because I didn't want to deal with it, and it's unrelated to what I am doing, but this route already has to access the application controller so perhaps it could have just inlined what the mixin is doing